### DISCUSSION TITLE
How to remove console log info when enviroment is production?

### DISCUSSION THREAD

**Discussion by KahrLu** [nuxt/nuxt#16464] [Questions]
@pi0 , At present, nuxt3 will output every api request to the console, which enables us to generate a lot of unnecessary log records when collecting console logs. How do I remove it?

**Comment by pi0** [comment]
Hi @KahrLu. What do you mean by "output every api request to the console". Normally not production build itself should not log each request to server console. Are you thinking to hide all custom console.logs inside code for production?

**Reply by JiaZhiheng** [reply]
I want have this question's answer，do you have plugin to delete “console.log”
